draft-ietf-core-dns-over-coap.md

```diff
@@ -499,6 +507,11 @@ harden against injecting spoofed responses.
 Consequently, it is of little concern to leverage the benefits of CoAP caching by setting the ID to
 0.
 
+General CoAP security considerations apply.
+Exceeding those in {{Section 11 of RFC7252}},
+the request patterns of DoC make it likely that long-lived security contexts are maintained:
+{{amp-0rtt}} goes into more detail on what can and needs to be done
+when those are resumed from a new address.
```

Comment:
```suggestion
when those are resumed from a new source address or port.
```

Reply:
Maybe 'new endpoint' – source-address-and-port is a UDP/TCP thing. Thinking of OSCORE, you're also in a "new endpoint" situation if after requests over TCP from an address/port combination, all of a sudden the requests come from the same UDP address/port. (One might argue that it's very likely that this is return routable if TCP was, the same argument can also be made for same-IP-different-port, and then we'd have to think about NAT, and I don't want to think about NAT).

Reply:
Updated to "new endpoint" – that fine?